src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -118,7 +118,7 @@ class PrettyPageHandler extends Handler
         "macvim"   => "mvim://open/?url=file://%file&line=%line",
         "phpstorm" => "phpstorm://open?file=%file&line=%line",
         "idea"     => "idea://open?file=%file&line=%line",
-        "vscode"   => "vscode://file/%file:%line",
+        "vscode"   => "vscode://file%file:%line",
```

Comment:
This seems possibly not correct. The new format is like this only on non-Windows. On Windows we still need a leading slash in the format. ([Possible update to VS Code docs](https://github.com/microsoft/vscode-docs/pull/6838/files))

We would need to introduce additional processing here: a conditional, or at least a str_replace. This is inconvenient. :-(